pkg/utils/utils.go

```diff
@@ -66,6 +66,12 @@ var (
 	errInvalidSpaceReference = errors.New("invalid storage space reference")
 )
 
+// ShareMetadata contains Metadata for a share
+type ShareMetadata struct {
```

Comment:
Could you move this to the `share` package and then rename it to just `Metadata`?

Reply:
:+1: